Cargo.toml

```
@@ -17,13 +17,16 @@ failure = "0.1.2"
 memmap = "0.7.0"
 murmurhash3 = "~0.0.5"
 ndarray = "0.12"
-needletail = "0.2.2"
+needletail = { path = "../needletail/" }
```

you can do a git branch with the current PR in needletail, if you want:
```suggestion
needletail = { git = "https://github.com/onecodex/needletail", branch = " v0.3" }
```

---

Just force pushed that in. Thanks!